.changeset/sharp-oranges-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/onchainkit": patch
+---
+
+**feat**: [Swap] ERC-20 Approval Flow by @0xAlec #761
```

Comment:
So the fact we are removing `onSubmit` will create a breaking changes.

We need to make sure we document this in the changeset. Like you see in some cases in the CHANGELOG https://github.com/coinbase/onchainkit/blob/main/CHANGELOG.md

Reply:
added to changelog (but not under a version since we haven't released it yet - will let you take that on)